src/main/java/core/basesyntax/DoFigure.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax;
+
+public interface DoFigure {

```

Comment:
The interface name `DoFigure` is not following the Java naming conventions for interfaces. According to the checklist, avoid using verbs for class/interface names. Consider renaming it to something more indicative of its purpose, such as `FigureOperations` or `DrawableFigure`.